solana/programs/token-bridge-relayer/build.rs

```diff
@@ -34,19 +30,12 @@ fn main() -> Result<()> {
 #[derive(Deserialize)]
 #[serde(rename_all = "camelCase")]
 struct Network {
-    #[cfg(any(feature = "mainnet", not(feature = "testnet")))]
-    mainnet: NetworkAddresses,
+    #[cfg(feature = "mainnet")]
+    mainnet: String,
     #[cfg(feature = "testnet")]
     testnet: NetworkAddresses,
```

Comment:
This should be a `String` too, right?

Reply:
Yes, it's fixed. Thanks for catching this.